cob_android_msgs/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ catkin_package(
 #  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 #)
 
-install(DIRECTORY msg
+install(DIRECTORY srv
```

Comment:
we do not install any msg/srv/action folder....installing the generated message header is handled by catkin....